examples/with-portals/components/Portal.js

```diff
@@ -4,13 +4,10 @@ import ReactDOM from 'react-dom'
 export class Portal extends React.Component {
   componentDidMount () {
     this.element = document.querySelector(this.props.selector)
+    this.forceUpdate()
   }
 
   render () {
-    if (this.element === undefined) {
```

Comment:
This is still needed for SSR

Reply:
Basically only adding `this.forceUpdate` is enough

Reply:
done